### ISSUE TITLE
Should some "Other" options  in some pages be changed to text area

### ISSUE THREAD

**Issue by lmcbhvu** [cds-snc/report-a-cybercrime#1442]
## Summary

Should some "Other" options  in some pages be changed to text area instead of input box.  The user cannot see the  whole text that he/she enters if it is too long. 

## Steps to reproduce

Please see the picture

![error4](https://user-images.githubusercontent.com/60269104/74842314-34d59100-52f8-11ea-92ef-0562f5f1d901.jpg)

## Unresolved questions

> Are there any related issues you consider out of scope for this issue that could be addressed in the future?

**Comment by DianeLiu2019** [comment]
we decided we don’t need the box because the user can enter the data on the next screen.
